source/main/index.js

```diff
@@ -162,8 +166,23 @@ const onAppReady = async () => {
   );
   saveWindowBoundsOnSizeAndPositionChange(mainWindow, requestElectronStore);
 
-  logger.info('Setting up Cardano Node...');
-  cardanoNode = setupCardanoNode(launcherConfig, mainWindow);
+  const getRtsFlagsAccordingToSpecs = () => {
```

Comment:
Just a question:
Is it always based on specs? If `getRtsFlags(network)` might return something different then it's not always by specs, but I'm not aware where do we set those values in electron store etc.

Reply:
`getRtsFlagsAccordingToSpecs` return the flags are stored in the local-storage. If nothing has been defined, the recommended flag according to your PC specs is returned. Do you have a suggestion for the name @przemyslaw-wlodek ?